src/Behat/Testwork/PathOptions/Printer/ConfigurablePathPrinter.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Behat Testwork.
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Behat\Testwork\PathOptions\Printer;
+
+final class ConfigurablePathPrinter
```

Comment:
This is the new service that prints paths, adding an option to decide if they should be printed as relative (default) or absolute